### ISSUE TITLE
Many to one  Cannot add new items

### ISSUE THREAD

**Issue by cdwmhcc** [directus/api#1121]
# Bug Report
Many to one  Cannot add new items

## Steps to Reproduce

1. Create a collection blog, add field title, content
2. Create a collection blog_like and add a field `blogid` (many to one)
3. Setting Public Permissions
4. Call the interface to add like
 ```
this.$client.createItem("blog_like",{blogid: 1}).then((res) => {
   console.log(res)
 })
```
4.See error
> Statement could not be executed (23000 - 1048 - Column 'action_by' cannot be null)

![image](https://user-images.githubusercontent.com/8030134/61401022-20fe2280-a904-11e9-908c-4f229a20b633.png)

## Expected Behavior
Many to one  Cannot add new entries

## Actual Behavior
It used to work, but I don't know why it suddenly stopped working.
Maybe because of the update of jsSDK, maybe the problem of api, maybe the problem of app. 
Hope to be fix

## Technical Details

- Device: [Desktop]
- OS: [Windows 10.1809]
- Web Server: [Nginx:1.17.0]
- PHP Version: [7.3.6]
- Database: [5.7.26]
- API Version: [2.3.0]
- APP Version: [7.7]
- Install Method: [cloned `master` branch]

**Comment by bjgajjar** [comment]
@cdwmhcc 

May I have a dump SQL file?

**Comment by bjgajjar** [comment]
Fixed in #1126

**Comment by cdwmhcc** [comment]
> Fixed in #1126

Thank you!
